app/models/conference/importer.rb

```diff
@@ -57,17 +63,12 @@ def process_csv(file)
             url: row['Website'],
             notes: row['Notes'],
           }
-          conference.update!(conference_hash.merge(season_id: fetch_season_id(row['UID'])))
+          conference.update!(conference_hash)
```

Comment:
I 'undid' merging because of the spec :
```it 'assign the season_id' do
        FactoryGirl.create(:conference, gid: 2017001, season_id: nil)
        FactoryGirl.create(:conference, gid: 2018005, season_id: nil)
        s_2017 = FactoryGirl.create(:season, name: "2017")
        s_2018 = FactoryGirl.create(:season, name: "2018")

        expect{subject}.to change{Conference.find_by(gid: 2017001).season_id}.to(s_2017.id)
        expect(Conference.find_by(gid: 2018005).season_id).to eq s_2018.id
      end
```
I did not find a way to check the change in the specs as well as merging in the season_id. 
I understand that is a bad reason to do an extra db action, but I also really want to doublecheck if the season_id has been changed.
Is there a way to do that? Or should I only care that the conference has the correct season_id?